Dynasty_Warriors_3_Playstation_2/Dynasty_Warriors_3.rascript

```diff
@@ -4,7 +4,7 @@
 // NOTE all code notes may change, please check RA for the latest code notes. 
 
 onePlayerString = " [1P]"
-asASinglePlayerString = " as a single player"
+asASinglePlayerString = " as a single player."
```

Comment:
Can you please remove the punctuation changes (At least the periods at the end of Descriptions).  I appreciate the thought, but it's not required (as long as it's uniformed) and not prefered. 

Reference:
https://docs.retroachievements.org/guidelines/content/writing-policy.html#punctuation-1